test/ERC20RootVault.test.ts

```diff
@@ -0,0 +1,1149 @@
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [0, randomInt(100)]
+                                        )
+                                ).to.be.revertedWith(Exceptions.VALUE_ZERO);
+                            }
+                        );
+                    });
+                });
+
+                describe("when owner of subvaultNft is not a contract", () => {
+                    it(`reverts with ${Exceptions.FORBIDDEN}`, async () => {
+                        const startNft = (
+                            await this.vaultRegistry.vaultsCount()
+                        ).toNumber();
+                        const newOwner = randomAddress();
+                        await withSigner(
+                            await this.vaultRegistry.ownerOf(startNft),
+                            async (signer) => {
+                                await this.vaultRegistry
+                                    .connect(signer)
+                                    .setApprovalForAll(newOwner, true);
+                                await this.vaultRegistry
+                                    .connect(signer)
+                                    .transferFrom(
+                                        signer.address,
+                                        newOwner,
+                                        startNft
+                                    );
+                            }
+                        );
+
+                        await withSigner(
+                            this.erc20RootVaultGovernance.address,
+                            async (signer) => {
+                                await expect(
+                                    this.subject
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [startNft]
+                                        )
+                                ).to.be.revertedWith(Exceptions.FORBIDDEN);
+                            }
+                        );
+                    });
+                });
+
+                describe("when subvaultNft index is 0 (Somehow works)", () => {
```

Comment:
in our deployment it means that rootVault has itself as subVault, because we deploy rootVault firstly => it gets 0 nft.
So remove (somehow works) and replace it with the following explanation or smth pls